build/utils/withName.js

```diff
@@ -0,0 +1,5 @@
+module.exports = (name) => (fn) => {
+  const f = (...args) => fn(...args);
+  f.displayName = name;
+  return f;
+};
```

Comment:
@eltongarbin 
Just a JS best practice doubt. 
Is it a best practice separate these "functions" one per file?

Reply:
I don't think so.
It's more personal preference ;)